styles/main.css

```diff
@@ -0,0 +1,109 @@
+body {
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  padding-left: 0.938rem;
+  padding-right: 0.938rem;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+@media (min-width: 768px) {
+  .container {
+    width: 750px;
+  }
+}
+
+@media (min-width: 992px) {
+  .container {
+    width: 970px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .container {
+    width: 1170px;
+  }
+}
+
+header .container {
+  display: flex;
+  justify-content: flex-end;
+  padding: 0.938rem;
+}
+
+header .container .icon {
+  display: flex;
+  flex-flow: column nowrap;
+  width: 1rem;
+  gap: 0.25rem;
+  margin-right: 1.188rem;
+}
+
+header .container .icon span {
+  flex-basis: 100%;
+  border-bottom: 0.125rem solid black;
+}
+
+main {
+  background-image: url('../assets/header-illsutration-mobile@2x.png');
+
+  /* background-size: cover; */
+  background-repeat: no-repeat;
+  background-size: 100% cover;
+  height: 1000px;
+}
```

Comment:
Kindly update your headline section to match the figma design better🙏

Your design: 
<img width="236" alt="image" src="https://user-images.githubusercontent.com/37116322/234968324-247beec7-0ca7-4cc6-a6cc-402444084976.png">

Figma design:
<img width="210" alt="image" src="https://user-images.githubusercontent.com/37116322/234968395-5b403cb5-3ac8-4614-b8e6-429d59d4803e.png">

A few things to note:
- Please reposition and resize your background image so that it matches the figma design
- Please increase the spacing between the list of social icons and the arrow icon below them
- Please reduce the spacing between the subtitle and the paragraph text